.rubocop.yml

```diff
@@ -37,4 +37,4 @@ Naming/UncommunicativeMethodParamName:
   - id
 
 RSpec/NestedGroups:
-  MaxNesting: 4
```

Comment:
wut really? I added that line and rubocop didn't complain about it!

Reply:
I thought that in the rubocop docs I had read it was `MaxNesting`, not `Max`.. oh well :)

Reply:
`MaxNesting` is deprecated